src/vector-crypto.adoc

```diff
@@ -269,6 +269,17 @@ Element Group Size (`EGS`).
 LMUL constraints::
 For element-group instructions, `LMUL`*`VLEN` must always be at least as large as `EGW`, otherwise an
 _illegal instruction exception_ is raised, even if `vl`=0.
++
+A *Vector Element Group* operand or destination sets `EMUL = LMUL`.
++
+A *Scalar Element Group* operand or destination sets `EMUL = ceil(EGW / VLEN)`.
++
+The standard RISC-V Vector mandates on vector register group indices alignment to `EMUL` apply: operand or destination register indices must be aligned to `EMUL`, all other non-aligned cases are __reserved__.
++
+[NOTE]
+====
+These `EMUL` alignement constraints allow scalar element group operands to be allocated to any vector register group (without need to be aligned to `LMUL`) for any implementation with `VLEN >= EGW`.
```

Comment:
"register group" => "register", right?  and I think delete the "without need [...]" parenthetical.